torch/csrc/jit/passes/onnx.cpp

```diff
@@ -278,6 +285,8 @@ void NodeToONNX(
               new_block->owningGraph()->create(c10::onnx::Constant);
           const_node->t_(attr::value, value);
           const_node->output()->setType(TensorType::create(value));
+          UpdateReliable(
```

Comment:
we call `ONNXShapeTypeInference` for this const_node a few lines later. do we need to UpdateReliable explicitly here?

Reply:
Don't need, just removed.